### ISSUE TITLE
Error with takeScreenshot

### ISSUE THREAD

**Issue by dgarciabriseno** [Helioviewer-Project/api#177]
Command failing:
`kdu_expand -i 2022_07_12__00_00_05_129__SDO_AIA_AIA_304.jp2 -o result.pgm -reduce 11 -region "{0,0},{1,1}"`

Result:
```
Kakadu Core Error:
Attempting to access a non-existent resolution level within some
tile-component.  Problem almost certainly caused by trying to discard more
resolution levels than the number of DWT levels used to compress a
tile-component.
```

**Comment by dgarciabriseno** [comment]
I see a few more of these in the log, but I'm unable to reproduce them in helioviewer.
Helioviewer doesn't let you zoom out enough to reach an `imageScale` that results in `-reduce 11`
Need to look closer at where these requests are coming from.

**Comment by dgarciabriseno** [comment]
I found that I can use `kdu_expand -record` to read how many DWT levels exist in the file, this can be used to limit `-reduce` so that this error doesn't occur.

**Comment by wafels** [comment]
Did you find out how `-reduce 11 `might even happen in the normal course of hv operations?

**Comment by dgarciabriseno** [comment]
I have a theory, but I don't know how or why anyone is trying to take screenshots with such zoomed out image scales.
Image scale comes from the scale of the viewport vs. the sun.
I tested that if I make my helioviewer.org window small, imageScale will decrease, and if I maximize it then imageScale increases.

So the theory is fully zoomed out on a high resolution screen could result in strange image scales. I think that would make it happen, but I don't know why anyone would do that.

**Comment by dgarciabriseno** [comment]
Fixed by #185
